src/css/components/menu.less

```diff
@@ -0,0 +1,43 @@
+[role="menu"] {
```

Comment:
As you should specify role any way (e.g. to support screen reader), there is no need to add an additional class.